src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,8 +1,102 @@
-import React from 'react'
+import React, { useTransition } from 'react'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faChevronLeft,
+  faChevronRight,
+  faCartShopping,
+} from '@fortawesome/free-solid-svg-icons'
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
```

Comment:
```suggestion
import React, { useTransition, useState } from 'react'
import { Link } from 'react-router-dom'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import {
  faChevronLeft,
  faChevronRight,
  faCartShopping,
} from '@fortawesome/free-solid-svg-icons'
```
import 순서 수정해주세요! 일반적인 convention을 따라 순서만 잘 지켜주셔도 가독성이 좋아집니다. 
아래 순서 참고해주세요.

React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)

Reply:
그리고 useTransition은 사용하지 않는 부분이라고 보여지네요! 제거해주세요